backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/Wav.java

```diff
@@ -61,7 +61,12 @@ public Sound (OpenALLwjglAudio audio, FileHandle file) {
 			WavInputStream input = null;
 			try {
 				input = new WavInputStream(file);
-				setup(StreamUtils.copyStreamToByteArray(input, input.dataRemaining), input.channels, input.sampleRate);
+				if (input.type == 0x0055) {
+					setType("mp3");
```

Comment:
This is supporting MP3 inside WAV? It doesn't seem necessary to support. Why would someone do this?

Reply:
This practice was more common in older software and possibly hardware than it is today. Perfectly valid - just a bit silly when you can use the proper container and extension instead.

If we'd rather not support MP3 WAVs, more code than what you've highlighted here can be reverted (you probably already know this - just a note for everyone).

Here is one reason why one may go out of their way to put MP3 in a WAV, albeit in related to libGDX: http://www.studiodust.com/riffmp3.html#what

Sound Recorder for Windows 95 to XP used the WAV container for MP3, as did anything else that relied on ACM for encoding audio. It's a truly pants program. 
https://microsoft.fandom.com/wiki/Sound_Recorder#Features